meta-processor/build.gradle

```diff
@@ -0,0 +1,12 @@
+apply plugin: 'kotlin'
+
+dependencies {
+  implementation "com.squareup:kotlinpoet:1.12.0"
```

Comment:
can you move the dependencies to the dependency definition to keep them in sync? that way we don't actually have to include this module in the dependabot config